hypha/apply/projects/models/payment.py

```diff
@@ -183,6 +183,13 @@ class Invoice(models.Model):
         verbose_name=_("Invoice amount"),
     )
     status = FSMField(default=SUBMITTED, choices=INVOICE_STATUS_CHOICES)
+    approved_by = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        on_delete=models.SET_NULL,
+        null=True,
+        blank=True,
+    )
+    approved_at = models.DateTimeField(null=True, blank=True)
```

Comment:
Can we rename it to `final_approval_...` or something that doesn't confuse it with internal approvals like staff approval or finance approval(in case of extended workflow)? Because we have to stamp the pdf only with final approval(finance approval or finance2 approval).

Or we may directly look out for the final approval activity as the [activity](https://github.com/HyphaApp/hypha/blob/main/hypha/apply/activity/models.py#L98) already has the date and user.